blueocean-web/src/main/js/main.jsx

```diff
@@ -37,14 +37,15 @@ class App extends Component {
     }
 
     render() {
+        const renderAdminLink = AppConfig.getInitialUser() !== "anonymous";
```

Comment:
Well, it wasnt in scope of ticket, and I don't have a good way currently to do this. We need permissions stuff really before we do this. I could hack it probabl though if it is critical.